ports/rbdl/portfile.cmake

```
@@ -0,0 +1,105 @@
+# Common Ambient Variables:
+#   CURRENT_BUILDTREES_DIR    = ${VCPKG_ROOT_DIR}\buildtrees\${PORT}
+#   CURRENT_PACKAGES_DIR      = ${VCPKG_ROOT_DIR}\packages\${PORT}_${TARGET_TRIPLET}
+#   CURRENT_PORT_DIR          = ${VCPKG_ROOT_DIR}\ports\${PORT}
+#   CURRENT_INSTALLED_DIR     = ${VCPKG_ROOT_DIR}\installed\${TRIPLET}
+#   DOWNLOADS                 = ${VCPKG_ROOT_DIR}\downloads
+#   PORT                      = current port name (zlib, etc)
+#   TARGET_TRIPLET            = current triplet (x86-windows, x64-windows-static, etc)
+#   VCPKG_CRT_LINKAGE         = C runtime linkage type (static, dynamic)
+#   VCPKG_LIBRARY_LINKAGE     = target library linkage type (static, dynamic)
+#   VCPKG_ROOT_DIR            = <C:\path\to\current\vcpkg>
+#   VCPKG_TARGET_ARCHITECTURE = target architecture (x64, x86, arm)
+#   VCPKG_TOOLCHAIN           = ON OFF
+#   TRIPLET_SYSTEM_ARCH       = arm x86 x64
+#   BUILD_ARCH                = "Win32" "x64" "ARM"
+#   MSBUILD_PLATFORM          = "Win32"/"x64"/${TRIPLET_SYSTEM_ARCH}
+#   DEBUG_CONFIG              = "Debug Static" "Debug Dll"
+#   RELEASE_CONFIG            = "Release Static"" "Release DLL"
+#   VCPKG_TARGET_IS_WINDOWS
+#   VCPKG_TARGET_IS_UWP
+#   VCPKG_TARGET_IS_LINUX
+#   VCPKG_TARGET_IS_OSX
+#   VCPKG_TARGET_IS_FREEBSD
+#   VCPKG_TARGET_IS_ANDROID
+#   VCPKG_TARGET_IS_MINGW
+#   VCPKG_TARGET_EXECUTABLE_SUFFIX
+#   VCPKG_TARGET_STATIC_LIBRARY_SUFFIX
+#   VCPKG_TARGET_SHARED_LIBRARY_SUFFIX
+#
+# 	See additional helpful variables in /docs/maintainers/vcpkg_common_definitions.md
+
+# # Specifies if the port install should fail immediately given a condition
+# vcpkg_fail_port_install(MESSAGE "rbdl currently only supports Linux and Mac platforms" ON_TARGET "Windows")
+
+set(VERSION 2.6.0)
+
+if (VCPKG_LIBRARY_LINKAGE STREQUAL static)
+    set(RBDL_STATIC ON)
+else()
+    set(RBDL_STATIC OFF)
+endif()
+
+vcpkg_download_distfile(ARCHIVE
+    URLS "https://github.com/rbdl/rbdl/archive/v2.6.0.tar.gz"
+    FILENAME "rbdl260.tar.gz"
+    SHA512 7b5fd03c0090277f295a28a1ff0542cd8cff76dda4379b3edc61ca3d868bf77d8b4882f81865fdffd0cf756c613fe55238b29a83bc163fc32aa94aa9d5781480
+)
+
+vcpkg_extract_source_archive_ex(
+    OUT_SOURCE_PATH SOURCE_PATH
+    ARCHIVE ${ARCHIVE}
+    # (Optional) A friendly name to use instead of the filename of the archive (e.g.: a version number or tag).
+    REF ${VERSION}
+    # (Optional) Read the docs for how to generate patches at:
+    # https://github.com/Microsoft/vcpkg/blob/master/docs/examples/patching.md
+    PATCHES
+        001_x64_number_of_sections_exceeded_in_object_file_patch.diff
+)

```

```suggestion
vcpkg_from_github(ARCHIVE
    OUT_SOURCE_PATH SOURCE_PATH
    REPO rbdl/rbdl
    REF v2.6.0
    SHA512 7b5fd03c0090277f295a28a1ff0542cd8cff76dda4379b3edc61ca3d868bf77d8b4882f81865fdffd0cf756c613fe55238b29a83bc163fc32aa94aa9d5781480
    HEAD_REF master
    PATCHES 001_x64_number_of_sections_exceeded_in_object_file_patch.diff
)
```
Since the source codes can be downloaded from GitHub, we prefer to use `vcpkg_from_github()` to get the source.

---

I didn't know `vcpkg_from_github()`. Thank you for the information! 😉

I followed [Packaging Zipfiles guide](https://vcpkg.readthedocs.io/en/latest/examples/packaging-zipfiles) and I filled the template generated by `vcpkg create` command.